File: static/algorithms/sorting/insertion_sort/go/insertion_sort.go
package main

import "fmt"

func insertionSort(items []int) {
	for i := range items {
		c := i
		inserting := items[i+1]
		for ; (items[c] > inserting) && (c >= 0); c-- {
			fmt.Println(c)
			items[c+1] = items[c]
		}
		items[c+1] = inserting
	}
}

func main() {
	items := []int{102, 532, 123, 489}
	insertionSort(items)
	fmt.Println(items)
}

File: static/algorithms/sorting/insertion_sort/py/insertion_sort.py
def insertion_sort(l):
    for i in range(1, len(l)):
        j = i-1
        key = l[i]
        while (l[j] > key) and (j >= 0):
           l[j+1] = l[j]
           j -= 1
        l[j+1] = key

numbers = [10, 9, 8, 7, 6, 5, 4, 3, 2, 1]
insertion_sort(numbers)
print(numbers)